proc_scripts/slice_FID_from_echo.py

```diff
@@ -2,8 +2,8 @@
 from pyspecdata import *
 from sympy import symbols
 
-def slice_FID_from_echo(s, ph1=1, ph2=-2):
-    fl = fl_mod() 
+
+def slice_FID_from_echo(s, max_t=None, ph1=1, ph2=-2, fl=None):
```

Comment:
I would actually say `pathway={'ph1':1, 'ph2':-2}` then below you can do something like this to select the appropriate coherence pathway (obviously replacing later occurrences of `s` with `myslice`
```
myslice = s
for k,v in pathway.items():
    myslice = s[k,v]
```